tests/data/test_parsers.py

```diff
@@ -0,0 +1,11 @@
+from telcell.data.parsers import parse_measurements_csv
+
+
+def test_parse_measurements_csv(path='./tests/testdata.csv'):
```

Comment:
Verwijzing naar relatief pad werkt niet altijd omdat working directory geen gegeven is. Zie https://stackoverflow.com/questions/32527861/python-unit-test-that-uses-an-external-data-file

Reply:
Een pytest fixture zou hier goed passen denk ik:

```python
@pytest.fixture
def testdata_path():
    return Path(__file__).parent / 'testdata.csv'
```

of iets langs die lijn, als het pad maar klopt :)

Reply:
Hoe de pytest fixture werkte snapte ik niet echt, misschien dat @akaIDIOT  dat nog kan aanpassen als dat belangrijk is. Ik snapte niet hoe de functie dan aangeroepen moest worden in de testfunctie, kreeg namelijk errors bij zowel testdata_path als testdata_path(). Bij de een werd er geklaagd dat ie een string/pathlike object wilde hebben ipv een fucntie en bij de ander dat je een pytest fixture niet direct mag aanroepen.

Reply:
Heb twee commits gepusht. Heb een fixture in `conftest.py` gezet omdat 'ie op meerdere plekken werd gebruikt (die file is een standaard-plek waar pytest gaat kijken). Een fixture aan een test functie koppelen doet pytest voor je, die plakt ze vooral op naam aan elkaar; een argument aan een testfunctie zal gevuld moeten worden voordat het werkt, dat doet pytest voor je door een fixture te zoeken met diezelfde naam. 

Met name ["Requesting Fixtures"](https://docs.pytest.org/en/6.2.x/fixture.html#requesting-fixtures) is denk ik de truc, al kan je er nog veel meer mee dan het simpele voorbeeld (hebben we hier nog even niet nodig mss, maar ze zijn best krachtig).

Reply:
Super thanks! dan lijkt deze klaar om te mergen :D